apps/builder/app/builder/features/workspace/canvas-tools/outline/editable-block-instance-outline.tsx

```diff
@@ -352,8 +356,8 @@ export const EditableBlockChildHoveredInstanceOutline = () => {
       <Flex
         css={{
           width: "min-content",
-          height: "min-content",
           pointerEvents: isMenuOpen ? "none" : "all",
+          clipPath: `polygon(0% 0%, 100% 0%, 100% 100%, 0% 24px)`,
```

Comment:
that 24px needs to be somehow reused from the values that define trigger size etc, otherwise if we change something there, there is no way we will not forget to update this value as well, because this thing is basically invisible